src/session/session.c

```diff
@@ -401,15 +400,7 @@ exit_init_problem (int result_code)
   else
     message = pam_strerror (NULL, result_code);
```

Comment:
Do we have a guarantee that PAM won't someday have a `\` or `"` in their messages?  I know this is an existing issue, so feel free to punt on it.

Reply:
Running out of time this morning, but good point. It's not a new problem, so I'd punt it for *this* PR. But I made a TODO entry for myself and also in #16808.

Reply:
Pushed here after all. Running out of razor blades!